CaloMC/src/CaloDigiMaker_module.cc

```diff
@@ -136,6 +136,7 @@ namespace mu2e {
       pulseShape_.buildShapes();
             
       noiseGenerator_.initialize(wfExtractor_); 
+noiseGenerator_.plotNoise("noise.pdf");
```

Comment:
was this a debug line?  It looks like it will always produce a file

Reply:
OOpsi, this is what happens when you code late being tired... Fixed

Reply:
@rlcee and @bechenard : I believe that the only outstanding work on this PR is for Bertrand to make the production of noise.pdf optional (or remove it).  Then for Ray to approve and then I merge.  Am I missing anything?